x/ibc/04-channel/types/msgs.go

```diff
@@ -100,7 +101,7 @@ func (msg MsgChannelOpenTry) ValidateBasic() error {
 	if len(msg.ProofInit) == 0 {
 		return sdkerrors.Wrap(commitmenttypes.ErrInvalidProof, "cannot submit an empty proof init")
 	}
-	if msg.ProofHeight == 0 {
+	if msg.ProofHeight.IsZero() {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidHeight, "proof height must be > 0")
```

Comment:
this comment applies for all Msgs that use `Height`. I think that the error messages need to be updated, does `EpochHeight` needs to be > 0?

Reply:
The core msgs just check that the Height isn't zero. Tendermint EpochHeights must start from 1, but this is specific to Tendermint semantics so those checks only happen for tm clients